File: PHP/Miscellaneous/GeneratedPrintAtHomeTicket.php
<?php
namespace LTD_API;


class GeneratedPrintAtHomeTicket
{
    //instance of PrintAtHomeTicketInfo
	private $ticket;
	//instance of int?
	private $fileSize;
	private $downloadUrl;
	
	public function getTicket()
    {
        return $this->ticket;
    }
    public function setTicket($ticket)
    {
        $this->ticket = $ticket;
    }

	public function getFileSize()
    {
        return $this->fileSize;
    }
    public function setFileSize($fileSize)
    {
        $this->fileSize = $fileSize;
    }

	public function getDownloadUrl()
    {
        return $this->downloadUrl;
    }
    public function setDownloadUrl($downloadUrl)
    {
        $this->downloadUrl = $downloadUrl;
    }
}

File: Java/Miscellaneous/GeneratedPrintAtHomeTicket.java
package LTD_API;

import java.util.List;

public class GeneratedPrintAtHomeTicket
{
    private PrintAtHomeTicketInfo ticket;
	private int? fileSize;
	private String downloadUrl;
	
	public PrintAtHomeTicketInfo getTicket()
    {
        return ticket;
    }
    public void setTicket(PrintAtHomeTicketInfo ticket)
    {
        this.ticket = ticket;
    }

	public int? getFileSize()
    {
        return fileSize;
    }
    public void setFileSize(int? fileSize)
    {
        this.fileSize = fileSize;
    }

	public String getDownloadUrl()
    {
        return downloadUrl;
    }
    public void setDownloadUrl(String downloadUrl)
    {
        this.downloadUrl = downloadUrl;
    }
}